webapi/_test_data/ontologies/beol-onto.ttl

```diff
@@ -149,6 +149,109 @@
                    rdfs:comment """Empfänger"""@de ,
                                 """Recipient"""@en .
 
+### ###########################################
+### beol:hasFigure
+
+:hasFigure rdf:type owl:ObjectProperty ;
+
+              rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+              rdfs:label "Zeichen"@de ,
+                         "Figure"@en ;
+
+              rdfs:comment """Zeichen von Brief"""@de ,
+                           """Figure of Letter"""@en ;
+
+              knora-base:objectClassConstraint :figure ;
+
+              salsah-gui:guiElement salsah-gui:Searchbox .
+
+### ###########################################
+### beol:hasFigureValue
+
+:hasFigureValue rdf:type owl:ObjectProperty ;
+
+                   knora-base:objectClassConstraint knora-base:LinkValue ;
+
+                   rdfs:subPropertyOf knora-base:hasLinkToValue ;
+
+                   rdfs:label "Zeichen"@de ,
+                              "Figure"@en ;
+
+                   rdfs:comment """Zeichen von Brief"""@de ,
+                                """Figure of Letter"""@en .
+
+### ###########################################
+### beol:letterHasTranslation
+
+:letterHasTranslation rdf:type owl:ObjectProperty ;
+
+              rdfs:subPropertyOf knora-base:hasLinkTo ;
+
+              rdfs:label "Übersetzung"@de ,
+                         "Translation"@en ;
+
+              rdfs:comment """Übersetzung"""@de ,
+                           """Translation"""@en ;
+
+              knora-base:subjectClassConstraint :letter ;
+
+              knora-base:objectClassConstraint :letter ;
+
+              salsah-gui:guiElement salsah-gui:Searchbox .
+
+### ###########################################
+### beol:letterHasTranslationValue
+
+:letterHasTranslationValue rdf:type owl:ObjectProperty ;
+
+                   knora-base:objectClassConstraint knora-base:LinkValue ;
+
+                   knora-base:subjectClassConstraint :writtenSource ;
```

Comment:
for `letterHasTranslation`, `knora-base:subjectClassConstraint` is `:writtenSource`